linux/files/collectd_df.conf

```diff
@@ -1,9 +1,10 @@
-LoadPlugin df
+<LoadPlugin df>
+  Globals false
+</LoadPlugin>
+
 <Plugin df>
   {%- for fs_type in plugin.get('fs_types', []) %}
   FSType {{ fs_type }}
   {%- endfor %}
-  {%- if plugin.get('ignore_selected', False) %}
-  IgnoreSelected true
-  {%- endif %}
+  IgnoreSelected {{ plugin.get('ignore_selected', true)|lower }}
```

Comment:
s/true/True/?

Also it means that you invert the behavior when 'ignore_selected' isn't defined. Is that ok?